server.js

```diff
@@ -117,6 +117,21 @@ app.post('/api/delete_record', function(req, res){
   })
 })
 
+// route for pledge
+app.post('/api/make_pledge', function(req, res){
+  const {broadcaster_id, npo_id, dare_id, user_dare_id, pledger_id} = req.body;
+  var pledgeAmount = req.body.pledgeAmount
+  var queryString = "INSTERT INTO pledge (broadcaster_id, npo_id, dare_id, user_dare_id, pledger_id, pledgeAmount) " + 
+                    "VALUES (" + broadcaster_id + "," npo_id "," + dare_id + "," + user_dare_id + "," + pledger_id ", " + pledgeAmount ")" 
```

Comment:
Use a query string. Also "INSTERT" won't work. Write some tests for this route (see morgan)